test/testinfra/MUXTestInfra/TestAppInstallHelper.cs

```diff
@@ -139,9 +147,9 @@ public static void InstallCert(string certFilePath)
         /// <summary>
         /// Installs the cert file for the appx so that it can be installed on the desktop environment
         /// </summary>
-        public static void InstallAppxCert(string deploymentDir, string packageName)
+        public static void InstallAppxCert(string deploymentDir, string certFileName)
         {
-            InstallCert(Path.Combine(deploymentDir, packageName + ".cer"));
```

Comment:
I think this may have broke the release pipeline because the cert file wasn't found